modules/nw-metallb-bgppeer-cr.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-configure-bgp-peers.adoc
+
+[id="nw-metallb-bgppeer-cr_{context}"]
+= About the BGP peer custom resource
+
+The fields for the BGP peer custom resource are described in the following table.
+
+.MetalLB BGP peer custom resource
+[cols="1,1,3",options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`metadata.name`
+|`string`
+|Specifies the name for the BGP peer custom resource.
+
+|`metadata.namespace`
+|`string`
+|Specifies the namespace for the BGP peer custom resource.
+
+|`spec.myASN`
+|`integer`
+|Specifies the Autonomous System number for the local end of the BGP session.
+The range is `0` to `65535`.
+
+|`spec.peerASN`
+|`integer`
+|Specifies the Autonomous System number for the remote end of the BGP session.
+The range is `0` to `65535`.
+
+|`spec.peerAddress`
+|`string`
+|Specifies the IP address of the peer to contact for establishing the BGP session.
+
+|`spec.sourceAddress`
+|`string`
+|Optional: Specifies the IP address to use when establishing the BGP session.
+The value must be an IPv4 address.
+
+|`spec.peerPort`
+|`integer`
+|Optional: Specifies the network port of the peer to contact for establishing the BGP session.
+The range is `0` to `16384`.
+
+|`spec.holdTime`
+|`string`
+|Optional: Specifies the duration for the hold time to propose to the BGP peer.
+The minimum value is 3 seconds (`3s`).
+The common units are seconds and minutes, such as `3s`, `1m`, and `5m30s`.
+To detect path failures more quickly, also configure BFD.
+
+|`spec.keepaliveTime`
+|`string`
+|Optional: Specifies the maximum interval between sending keep-alive messages to the BGP peer.
+If you specify this field, you must also specify a value for the `holdTime` field.
+The specified value must be less than the value for the `holdTime` field.
+
+|`spec.routerID`
+|`string`
+|Optional: Specifies the router ID to advertise to the BGP peer.
+
+|`spec.password`
+|`string`
+|Optional: Specifies the MD5 password to send to the peer for routers that enforce TCP MD5 authenticated BGP sessions.
+
+|`spec.bfdProfile`
+|`string`
+|Optional: Specifies the name of a BFD profile.
+
+|`spec.nodeSelectors`
+|`object[]`
+|Optional: Specifies a selector, using match expressions and match labels, to control which nodes can connect to the BGP peer.
+
+|`spec.egpMultiHop`
+|`boolean`
+|Optional: Specifies that the BGP peer is multiple network hops away.
```

Comment:
I think we should emphasize that it relates only to EBGP peers (when ASN != MyASN)

Reply:
Agreed. That was a poor oversight on my part.  I added mention of "different AS" and a one-liner for eBGP.